<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>screenshot</class>
 <widget class="QDialog" name="screenshot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>329</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>393</width>
    <height>329</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>393</width>
    <height>329</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Take Screenshot</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/Pictures/wallpaper.png</normaloff>:/icons/Pictures/wallpaper.png</iconset>
  </property>
  <widget class="QCheckBox" name="hideThisWindowCheckBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>245</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Hide This Window</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="screenshotLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>251</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>354</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="newScreenshotButton">
      <property name="text">
       <string>New Screenshot</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveScreenshot">
      <property name="text">
       <string>Save Screenshot</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="QuitButton">
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>321</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Screenshot Delay:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="delaySpinBox"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
